src/pages/layer-2.js

```diff
@@ -0,0 +1,1099 @@
+          <Link to="/upgrades/vision/  ">Ethereum vision</Link>).
+        </p>
+        <p>
+          The Ethereum community has taken a strong stance that it would not
+          throw out decentralization or security in order to scale. Until
+          sharding, Ethereum Mainnet (layer 1) will only be able to process{" "}
+          <Link to="https://ethtps.info/Network/Ethereum">
+            roughly 15 transactions per second
+          </Link>
+          . When demand to use Ethereum is high this causes network congestion,
+          increasing transaction fees, and pricing out those who cannot afford
+          it from using Ethereum until the fees reduce. That is where layer 2
+          comes in to scale Ethereum today.
+        </p>
+
+        <h3>Benefits</h3>
+        <InfoGrid>
+          {layer2Cards.map(({ emoji, title, description }, idx) => (
+            <Card
+              description={description}
+              title={title}
+              emoji={emoji}
+              key={idx}
+            />
+          ))}
+        </InfoGrid>
+      </LightGrayContent>
+
+      <PaddedContent id="what-is-layer-2">
+        <h2>What is layer 2</h2>
+        <p>
+          Layer 2 is a collective term for Ethereum scaling solutions that
+          handle transactions off Ethereum layer 1 while taking advantage of the
+          robust decentralized security model of Ethereum layer 1. A layer 2 is
+          a <b>separate blockchain that is connected to Ethereum</b>. It
+          regularly communicates with Ethereum (by submitting bundles of
+          transactions) in order to ensure it has similar security and
+          decentralization guarantees, and requires no changes to the layer 1
+          protocol. This lets layer 1 handle security, data availability, and
+          decentralization, whilst everything on the layer above (layer 2) can
+          handle scaling. Layer 2s takes the transactional burden away from the
+          layer 1, and posts finalized proofs back to the layer 1 to finalize
+          the state. By taking this load away from layer 1, the base layer will
+          become less congested, and everything becomes more scalable.
+        </p>
+        <h3>Rollups</h3>
+        <TwoColumnContent>
+          <FlexContainer flexPercent="60">
+            <p>
+              Rollups are currently the preferred layer 2 solution for scaling
+              Ethereum. By using rollups, users can{" "}
+              <Link to="https://l2fees.info/">
+                reduce gas fees by up to 100x
+              </Link>{" "}
+              when comparison to a layer 1 transaction.
+            </p>
+            <p>
+              Rollups bundle (or ’roll up’) hundreds of transactions into a
+              single transaction on layer 1. This allows the transaction fees to
+              be shared across everyone in the rollup. These bundled
+              transactions are posted to the layer 1 as validity, fraud, or
```

Comment:
Sentence `These bundled transactions are posted to the layer 1 as validity, fraud, or fault proofs.` can be confusing in this context and sounds wrong. Other sentences explain it nicely, I would go with something like: 
```
This allows the transaction fees to be shared across everyone in the rollup. Rollup transactions get executed outside of layer 1 but the transaction data gets posted to layer 1. This data are either proof of transaction validity in ZK rollups or can be used to create a fraud proof in case of optimistic rollup. Therefore by posting transaction data onto layer 1, rollups are able to inherit the security of Ethereum.
```